File: Web/RMA-Web-Shell/projects/fincare/src/app/billing-manager/models/bad-debt-reinstate.ts
import { TransactionTypeEnum } from "../../shared/enum/transactionTypeEnum";

export class BadDebtReinstate {
    amount: number;    
    transactionType: TransactionTypeEnum;
    transactionId:number;
    invoiceId:number;
    documentNumber:string;
    description: string;
  }

File: Service/Billing/RMA.Service.Billing.Contracts/Entities/BadDebtReinstate.cs
using RMA.Service.Admin.MasterDataManager.Contracts.Enums;

namespace RMA.Service.Billing.Contracts.Entities
{
    public class BadDebtReinstate
    {
        public decimal Amount { get; set; }
        public TransactionTypeEnum TransactionType { get; set; }
        public int TransactionId { get; set; }
        public int? InvoiceId { get; set; }
        public string DocumentNumber { get; set; }
        public string Description { get; set; }
    }
}
